lib/gui/app/modules/image-writer.js

```diff
@@ -126,16 +126,23 @@ exports.performWrite = (image, drives, onProgress) => {
       console.log(message)
     })
 
+    const flashResults = { errors: [] }
     ipc.server.on('fail', (error) => {
+      flashResults.errors.push(error)
```

Comment:
Isn't this information available through the flash results from the writer once it finished? Why do we need to track it again? The success page doesn't show until the flash results are in anyways, correct?